File: lat1dpbo_py/Mahasiswa.py
class Mahasiswa:
    __name = ""
    __nim = ""
    __prodigy = ""
    __faculty = ""
    
    def __init__(self, name = "", NIM = "", prodigy = "", faculty = ""):
        self.__name = name
        self.__nim = NIM
        self.__prodigy = prodigy
        self.__faculty = faculty
        
    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name
        
    def get_nim(self):
        return self.__nim
    
    def set_nim(self, NIM):
        self.__nim = NIM
    
    def get_prodigy(self):
        return self.__prodigy
    
    def set_prodigy(self, prodigy):
        self.__prodigy = prodigy
    
    def get_faculty(self):
        return self.__faculty
    
    def set_faculty(self, faculty):
        self.__faculty = faculty
    

File: lat1dpbo_php/Mahasiswa.php
<?php

class Mahasiswa {
    private $name = '';
    private $NIM = '';
    private $prodigy = '';
    private $faculty = '';

    public function __construct($name = '', $NIM = '', $prodigy = '', $faculty = ''){
        $this->name = $name;
        $this->NIM = $NIM;
        $this->prodigy = $prodigy;
        $this->faculty = $faculty;
    }

    public function setName($name){
        $this->name = $name;
    }

    public function getName(){
        return $this->name;
    }

    public function setNIM($NIM){
        $this->NIM = $NIM;
    }

    public function getNIM(){
        return $this->NIM;
    }

    public function setProdigy($prodigy){
        $this->Prodigy = $prodigy;
    }

    public function getProdigy(){
        return $this->prodigy;
    }

    public function setFaculty($faculty){
        $this->faculty = $faculty;
    }

    public function getFaculty(){
        return $this->faculty;
    }
}

?>